app/src/main/java/ch/heigvd/pro/b04/android/Network/RockinAPI.java

```diff
@@ -57,4 +59,14 @@
             @Path("idQuestion") String idQuestion,
             @Query("token") String token
     );
+
+    @PUT("/mod/{idModerator}/poll/{idPoll}/question/{idQuestion}/answer/{idAnswer}/vote")
+    Call<ResponseBody> voteForAnswer(
```

Comment:
Why do we have to specify a type of Call ? This request isn't supposed to return a value

Reply:
It returns a default response body, .enqueue() is impossible if the return value is null. We can either use this or Call<Void> as a return value.